lib/empty-example/sketch.js

```diff
@@ -1,7 +1,7 @@
 function setup() {
-  // put setup code here
+// put setup code here
 }
 
 function draw() {
-  // put drawing code here
```

Comment:
Can changes in this file be reverted as it is not related to the issue being addressed?

Reply:
ya let me change them.